sofl/main.py

```
@@ -47,6 +47,7 @@
 from sofl.importer.lutris_source import LutrisSource
 from sofl.importer.retroarch_source import RetroarchSource
 from sofl.importer.steam_source import SteamSource
+from sofl.importer.source import OnlineFixSource
```

Ruff (0.8.2)

`sofl.importer.source.OnlineFixSource` imported but unused

Remove unused import: `sofl.importer.source.OnlineFixSource`

(F401)